source/minio-cli/minio-kes-reference.rst

```diff
@@ -0,0 +1,759 @@
+authorization of clients making requests to the KES server. 
+
+Authentication
+~~~~~~~~~~~~~~
+   
+Both the client and server present their x.509 certificate and
+corresponding private key to their peer. The server only accepts the
+connection if the client certificate is valid and authentic:
+
+- A "valid" certificate is well-formed and current (i.e. not expired). 
+   
+  mTLS also mTLS also requires the following `Extended Key Usage
+  <https://tools.ietf.org/html/rfc5280#section-4.2.1.12>`__ extensions for 
+  the client and server. Specifically:
+
+  - The client certificate *must* include Client Authentication
+     (``extendedKeyUsage = clientAuth``)
+  
+  - The server certificate *must* include Server Authentication
+     (``extendedKeyUsage = serverAuth``)
+
+- An "authentic" certificate is issued by a trusted Certificate Authority
+  (CA). Both client and server *must* include the peer certificate CA 
+  in their local system trust store.
+
+  You can start the KES server with the :mc-cmd-option:`kes server auth`
+  option to perform mTLS with untrusted certificates during testing or early
+  development. However, untrusted certificates present a security
+  vulnerability and may open the KES server to access by unknown parties.
+  MinIO strongly recommends only allowing trusted certificates in production
+  environments.
+
+Once authenticated, the KES server proceeds to check if the client
+is *authorized* to perform the requested operation.
+
+.. _minio-kes-authorization:
+
+Authorization
+~~~~~~~~~~~~~
+   
+KES uses Policy-Based Access Control (PBAC) for determining what operations
+a given client has permission to perform. Each policy consists of
+one or more identities, where each identity corresponds to the
+SHA-256 hash of an x.509 certificate. The server only allows the
+client to perform the requested operation if *all* of the following
+are true:
+
+- A policy on the KES server contains the client identity.
+
+- The policy explicitly allows the requested operation.
+
+If no such policy exists on the KES server *or* if the policy does not
+explicitly allow the requested operation, the KES server denies the
+client's request. For more complete documentation on policies, 
+see :ref:`minio-kes-policy`.
+
+.. _minio-kes-policy:
+
+KES Policies
+~~~~~~~~~~~~
+
+KES uses policy-based access control (PBAC), where a policy describes the
+operations which an authenticated client may perform. 
+
+The following ``YAML`` document provides an example of the :kesconf:`policy`
+section of the KES server configuration document. The policy ``minio-sse`` 
+includes the appropriate :ref:`API endpoints <minio-kes-endpoints>` for 
+supporting MinIO Server-Side Encryption:
+
+.. code-block:: yaml
+   :class: copyable
+
+   policy:
+      minio-sse:
+         paths:
+         - /v1/key/create/*
+         - /v1/key/generate/*
+         - /v1/key/decrypt/*
+         - /v1/key/delete/*
```

Comment:
The ability to `delete` keys is not necessary and also not recommended since a compromised client (e.g. MinIO) could cause data loss.